### PR TITLE
Update my_worklist_approval.feature

### PR DESCRIPTION
**Pull request by garshol**
Simplified the use of Background and Given

### CODE REVIEW

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 1]
```diff
@@ -12,44 +12,37 @@ Feature: Curator handles an Approval Request
...
 
-    Rules:
-     - The User got publisher privilege
-     - User got publisher privilege, but all content need a Curator Approval to be publicly accessable
-     - Only the Curator got publisher privilege
-
 	Background: 
-		Given a User that needs an approval to publish
+		Given a User wants to publish a Resource
+		And a Institutional publication policy that needs an approval to publish
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:17]: an Institut…

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 2]
```diff
@@ -12,44 +12,37 @@ Feature: Curator handles an Approval Request
...
 		And the Curator may send an answer Message with a reason to the User
 
 	@3189
 	Scenario: Curator approves an Approval Request to grant open access to content
-		Given the User has published a Resource
-		And Institutional publication policy is "Registrator can only publish metadata"
+		Given Institutional publication policy is "Registrator can only publish metadata"
 		When the Curator approves the Request
 		Then the Request status is changed to "Approved"
 		And the the Resources' content is open accessable
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:40]: resource's

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 3]
```diff
@@ -12,44 +12,37 @@ Feature: Curator handles an Approval Request
...
 	@3189
 	Scenario: Curator approves an Approval Request to grant open access to content
-		Given the User has published a Resource
-		And Institutional publication policy is "Registrator can only publish metadata"
+		Given Institutional publication policy is "Registrator can only publish metadata"
 		When the Curator approves the Request
 		Then the Request status is changed to "Approved"
 		And the the Resources' content is open accessable 
-		# Don't remove the potentially original embargo on the files
+		And Don't remove the potentially original embargo on the files
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:41]: ???

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by terjehell — DISMISSED** [review]

File: 1223-curator_worklist/my_worklist_approval.feature (hunk 3 above)
Comment (garshol) [reply 1223-curator_worklist/my_worklist_approval.feature:41]: Content may be locked and have an embargo.
Only remove the lock on the content, not the embargo.

**Review by terjehell — DISMISSED** [review]

File: 1223-curator_worklist/my_worklist_approval.feature [hunk 4]
```diff
@@ -20,29 +20,29 @@ Feature: Curator handles an Approval Request
 
 	@3188
 	Scenario: Curator approves an Approval Request to publish
-		Given Institutional publication policy is "Only Curator can publish" 
+		Given an Institut publication policy is "Only Curator can publish"
```
Comment (brinxmat) [inline 1223-curator_worklist/my_worklist_approval.feature:23]: an Institutional

**Review by brinxmat — CHANGES_REQUESTED** [review]

**Review by terjehell — APPROVED** [review]

**Review by brinxmat — APPROVED** [review]

**Pull request merged**